packages/shared/src/hooks/squads/usePublicSquadRequests.ts

```diff
@@ -34,5 +73,29 @@ export const usePublicSquadRequests = (): UsePublicSquadRequestsResult => {
   return {
```

Comment:
Should we maybe memoize some of these?

Reply:
Good point. Most of the values here are stable. I think we can memoize the status so we won't have to instantiate and calculate days all the time.